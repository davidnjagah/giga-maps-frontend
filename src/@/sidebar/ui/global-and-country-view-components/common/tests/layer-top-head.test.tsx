import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import LayerTopHead from '../layer-top-head.view';

describe('Layer schools connectivity status', () => {
  it('should render component', () => {
    const { asFragment } = render(testWrapper(<LayerTopHead label="Data layer selection" />));
    expect(asFragment).toMatchSnapshot();
  });
});
