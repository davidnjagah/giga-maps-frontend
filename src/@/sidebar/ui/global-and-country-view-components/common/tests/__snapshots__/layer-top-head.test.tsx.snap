// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layer schools connectivity status should render component 1`] = `[Function]`;
